homeassistant/components/ecobee/climate.py

```diff
@@ -657,22 +681,6 @@ def set_auto_temp_hold(self, heat_temp, cool_temp):
 
         self.update_without_throttle = True
 
-    def set_fan_mode(self, fan_mode):
```

Comment:
This just moves the set_fan_mode routine out from in-between the set_temp routines.